packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -261,11 +261,10 @@ export interface ISegment extends IMergeNodeCommon, Partial<IRemovalInfo>, Parti
 	properties?: PropertySet;
 	addProperties(
```

Comment:
Fair suggestion, though I think the docs would just be `"Adds properties to this segment. See {@link Client.annotateRange} for additional context."`

I want this change to focus on ICombiningOp and related code and for it to not be blocked on docs changes, since I have other PRs blocked on this removal. I've opened https://github.com/microsoft/FluidFramework/pull/18317 which I think is a better place for docs changes to go.